rfc/0054-sdk3-management-apis.md

```diff
@@ -3070,6 +3072,46 @@ Nothing
 
     POST http://localhost:8091/pools/default/buckets/<bucket>/collections/<scope_name> -d name=<collection_name> -d maxTTL=<maxExpiry>
 
+## UpdateCollection
+
+Creates a new collection.
+
+### Signature
+
+```
+void UpdateCollection(CollectionSpec collection, [options])
+```
+
+### Parameters
+
+* Required:
+
+  * `collection`: [`CollectionSpec`](#collectionspec) - specification of the collection.
+
+* Optional:
+
+  * `Timeout` or `timeoutMillis` (`int`/`duration`) - the time allowed for the operation to be terminated. This is controlled by the client.
+
+  * `maxExpiry` (`duration`) - Value for the max expiry of new documents created without an expiry.
+
+### Returns
+
+Nothing
+
+### Throws
+
+* `InvalidArgumentsException`
+
+* `CollectionNotFoundException`
+
+* `ScopeNotFoundException`
+
+* Any exceptions raised by the underlying platform
```

Comment:
This isn't correct right?  As all exceptions are meant to derive from CouchbaseException in SDK3.

Reply:
I've just kept consistency with the rest of the RFC there.